skfda/exploratory/visualization/_baseplot.py

```diff
@@ -0,0 +1,82 @@
+"""BasePlot Module.
+
+This module contains the abstract class of which inherit all
+the visualization modules, containing the basic functionality
+common to all of them.
+"""
+
+from abc import ABC, abstractmethod
+from typing import List, Optional, Sequence, Union
+
+import matplotlib.pyplot as plt
+from matplotlib.artist import Artist
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+from ._utils import _figure_to_svg
+
+
+class BasePlot(ABC):
+    """
+    BasePlot class.
+
+    Attributes:
+        artists: List of Artist objects corresponding
+            to every instance of our plot. They will be used to modify
+            the visualization with interactivity and widgets.
+        fig: Figure over with the graphs are plotted.
+        axes: Sequence of axes where the graphs are plotted.
+    """
+
+    @abstractmethod
+    def __init__(
+        self,
+    ) -> None:
+        self.artists: List[Artist] = []
+        self.fig: Optional[Figure] = None
```

Comment:
Shouldn't `fig` and `axes` be init parameters?

Reply:
they can be init parameters, but they wont be used until set figure and axes is called

Reply:
Is there any plot where they won't be passed on init?

Reply:
no, it will be in everyone